src/realm/table.hpp

```diff
@@ -211,16 +211,25 @@ class Table {
 
     // Create an object with key. If the key is omitted, a key will be generated by the system
     Obj create_object(ObjKey key = {}, const FieldValues& = {});
-    // Create an object with specific GlobalKey.
+    // Create an object with specific GlobalKey - or return already existing object
+    // Potential tombstone will be resurrected
     Obj create_object(GlobalKey object_id, const FieldValues& = {});
     // Create an object with primary key - or return already existing object
+    // Potential tombstone will be resurrected
     Obj create_object_with_primary_key(const Mixed& primary_key, FieldValues&& = {});
     // Return key for existing object or return null key.
     ObjKey find_primary_key(Mixed value) const;
+    // Return ObjKey for object identified by id. If objects does not exist, return null key
+    ObjKey get_objkey(GlobalKey id) const;
     // Return key for existing object or return unresolved key.
     // Important: This is to be used ONLY by the Sync client. SDKs should NEVER
     // observe an unresolved key. Ever.
     ObjKey get_objkey_from_primary_key(const Mixed& primary_key);
+    // Return key for existing object or return unresolved key.
+    // Important: This is to be used ONLY by the Sync client. SDKs should NEVER
+    // observe an unresolved key. Ever.
+    // Important (2): This function must not be called for tables with primary keys.
```

Comment:
and if it is anyway, what's the reaction?

Reply:
Assertion

Reply:
ok